cmd/build.go

```diff
@@ -322,7 +344,8 @@ func copyBuildAndRuleFiles(moduleName, modulePath, buildFilesDir string, modules
 		packageName, targets := parseBuildFile(buildFile)
 		targetLines := []string{}
 		for _, targetName := range targets {
-			targetLines = append(targetLines, fmt.Sprintf("    ctx.AddTarget(reflect.TypeOf(__internal_pkg{}).PkgPath()+\"/%s\", %s)", targetName, targetName))
+			targetLines = append(targetLines,
+				fmt.Sprintf("    ctx.AddTarget(reflect.TypeOf(__internal_pkg{}).PkgPath()+\"/%s\", %s, out(\"\"))", targetName, targetName))
```

Comment:
`%q`?

Reply:
Does not work, unfortunately, since we need `"/xyz"`, not `"xyz"`